pkg/internalapi/conditions.go

```diff
@@ -18,3 +24,21 @@ const (
 	ProgressingReason       = "Progressing"
 	AwaitingConditionReason = "AwaitingCondition"
 )
+
+var (
+	DatacenterAvailableConditionFormat   = MakeDatacenterConditionFormat(scyllav1alpha1.AvailableCondition)
+	DatacenterProgressingConditionFormat = MakeDatacenterConditionFormat(scyllav1alpha1.ProgressingCondition)
+	DatacenterDegradedConditionFormat    = MakeDatacenterConditionFormat(scyllav1alpha1.DegradedCondition)
+)
+
+func MakeDatacenterConditionFormat(conditionType string) string {
+	return fmt.Sprintf("Datacenter%%s%s", conditionType)
```

Comment:
Something wrong with the format string here

Reply:
Okay - just realized that it's _indended_ for this function to return a format string. This was quite surprising to me reading this - can you make a comment to explain this intention?

```
// MakeDatacenterConditionFormat returns a parametric datacenter condition name, represented as a Sprintf format string.
```

Reply:
Information about it being a format string is already part of function signature. Isn't it clear enough?

Reply:
Not really - the fact that the result is expected to be consumed as a format string is not.

I mean, it can be reverse engineered by the reader if they notice that the `xxxFormat` naming convention is for `fmt` format strings. But a function returning a format string is unusual enough that imo it deserves a docstring.

Reply:
I have hard time understanding how comment about it returning format string helps reader if information about it is already in function signature.
Since this is such trivial thing I find pursuing further discussion while being under time pressure a waste of time. 
Done